src/test/scala/org/broadinstitute/dsde/firecloud/service/TSVFileSupportSpec.scala

```diff
@@ -154,6 +154,12 @@ class TSVFileSupportSpec extends AnyFreeSpec with TSVFileSupport {
     }
   }
 
+  List("1234d", "-5678D", "9012f", "3456F") foreach { str =>
```

Comment:
have you tried 123i? curious if it would treat i as "integer" or not. part of me wonders if just doing regex with number + letter should default to string, no matter what the letter is?

Reply:
I tried out 'd', 'f',' i' and 'l' to see what would happen, and 'd' and 'f' were the only ones that were coerced to numbers.